samples/artifacts/DataSummarization/NamedAggregation.siddhi

```diff
@@ -0,0 +1,67 @@
+@App:name("NamedAggregation")
+@App:description("Demonstrates how to create a named aggregation for better query performance.")
```

Comment:
Calculate aggregates for various time granularity

Reply:
0c298022e90d12e96fa96a6435c301871fc28edb This commit fixed the above correction